sdcm/cluster.py

```diff
@@ -1472,7 +1472,7 @@ def decode_backtrace(self):
             if self.termination_event.is_set() and self.test_config.DECODING_QUEUE.empty():
                 break
 
-    def copy_scylla_debug_info(self, node, debug_file):
+    def copy_scylla_debug_info(self, node: str, debug_file: str):
         """Copy scylla debug file from db-node to monitor-node
 
         Copy via builder
```

Comment:
method doc should be updated.
I'd also suggest to rename param `node` to `node_name`

Reply:
fixed